admin_manual/installation/nginx_configuration.rst

```diff
@@ -71,7 +67,11 @@ Nginx Configuration
       access_log off;
     }
 
-    location ~ ^/(?:\.htaccess|data|config|db_structure\.xml|README){
+    location ~* ^/(build|tests|config|lib|3rdparty|templates|data)/.* {
+      deny all;
+    }
+
+    location ~* ^/(\.user.ini|\.htaccess|\.mailmap|\.tag|autotest|occ|issue|indie|db_|console).* {
```

Comment:
I can't use 

``location ~* ^/(\.|autotest|occ|issue|indie|db_|console).* {``

here as this is also blocking the access to the .well-known URLs.